src/feature-management/FeatureFlagsProvider.tsx

```diff
@@ -26,6 +26,54 @@ export const FeatureFlagsProvider = ({children} : Props): React.ReactNode => {
 
     setFlagState({...flagState, loading: true})
 
+    /**
+     * CUSTOM PROPERTIES
+     *
+     * Custom properties allow you to target specific users or segments in CloudBees platform.
+     * Set these BEFORE calling Rox.setup() so they're available for targeting rules.
+     *
+     * Types:
+     * - setCustomStringProperty: For text values (userId, email, plan, etc.)
+     * - setCustomNumberProperty: For numeric values (age, score, count, etc.)
+     * - setCustomBooleanProperty: For true/false values (isPremium, isAdmin, etc.)
+     * - setCustomDateProperty: For date/time values (registrationDate, lastLogin, etc.)
+     *
+     * Use in CloudBees platform:
+     * Create targeting rules like:
+     * - "Show feature if userTier = 'premium'"
+     * - "Show feature if registrationDate > '2024-01-01'"
+     */
+
+    // String property example - User tier for subscription-based targeting
+    Rox.setCustomStringProperty('userTier', () => {
```

Comment:
I _think_ the `setCustomXXXProperty()` function lets you pass either a a value or a calculator function (need to check), and for these simple/fixed ones we might as well show the simple case, i.e. `setCustomStringProperty('userTier', 'premium')`.